test/unit/services/insights_plan_runner_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_plugin_helper'
+
+if defined? RedhatAccess
```

Comment:
perhaps you could just add `skip unless defined?(RedhatAccess)` to the test itself, but this also works fine